api_cgn_operator_search.yaml

```diff
@@ -29,6 +29,28 @@ paths:
           schema:
             $ref: "#/definitions/ProblemJson" 
 
+  "/search":
+    post:
+      operationId: search
+      summary: Search merchants or discounts
+      description: |
+        List of merchants and discounts with the token in name and/or description
+      parameters:
+        - in: body
+          name: body
+          required: false
+          schema:
+            $ref: '#/definitions/SearchRequest'
+      responses:
+        "200":
+          description: List of merchants/discounts for the given query parameters
+          schema:
+            $ref: "#/definitions/SearchResult"
+        "500":
```

Comment:
I'd add at lease the following response codes:

```typescript
        "400":
          description: Bad request
        "401":
          description: Token null or invalid.
```

Reply:
fixed in https://github.com/pagopa/io-backend/pull/1161/commits/ac49ea4395eae8b562fcb467639c955239f0996e

added just 400 and 401, 404 is not in this endpoint but is just present in the common type.